src/modding/ModManager.cs

```diff
@@ -754,26 +1099,67 @@ private void DisableAllPressed()
         UpdateOverallModButtons();
     }
 
+    private void EnableAllPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        if (enabledMods is null || notEnabledMods is null)
```

Comment:
Normal null checks should be used.
Also print an error as this seems like an error condition if this is ever true.